src/render.js

```diff
@@ -44,5 +46,5 @@ export function render(vnode, parentDom, replaceNode) {
  */
 export function hydrate(vnode, parentDom) {
 	parentDom._children = null;
-	render(vnode, parentDom);
+	render(vnode, parentDom, null);
```

Comment:
would it be possible to use Boolean `true` here? Semantically it might make a bit more sense, but also it avoids someone passing `null` values for replaceNode without realizing it - this is actually pretty likely, since DOM getters always return `null` for empty values (eg `foo.firstElementChild`).